Note about Unity UPM installation gotcha (#3068)
getsentry/sentry-docs: src/wizard/unity/index.md

Get the SDK via the [Unity Package Manager using a Git URL](https://docs.unity3d.com/Manual/upm-ui-giturl.html) to Sentry's SDK repository:

> > Make sure not empty space is appended to the end of the URL, otherwise Unity fails to find the package.

Author: I'm up for suggestions on how to phrase this and what format. I just to make sure no one will have the same bad experience I did just because of a bad copy and paste

Reviewer: submit a PR to UPM, url.trim() :D

> > Make sure not empty space is appended to the end of the URL, otherwise Unity fails to find the package.

Reviewer: ```suggestion
> Make sure trimming empty spaces appended to the end of the URL, otherwise, The Unity Package Manager fails to find the package.
```

not sure if any better, @PeloWriter

> > Make sure not empty space is appended to the end of the URL, otherwise Unity fails to find the package.

Reviewer 2: ```suggestion
> Confirm there's no empty space at the end of the line. The Unity Package Manager will fail to find the package if an empty space is appended to the end of the URL.
```
